src/platform/i386/vga.c

```diff
@@ -0,0 +1,204 @@
+#define ENABLE_VGA
```

Comment:
Actually I saw that
```
#define ENABLE_SERIAL
```
was done in platform/i386/kernel.c but I thought it isn't the right place and moved it to kernel.h instead. I'll look into cos_config.h and will make necessary changes for this. Thank you.